lib/auth0.js

```diff
@@ -12,7 +12,9 @@ const settings = {
     cookieSecret: process.env.SESSION_COOKIE_SECRET,
     cookieLifetime: process.env.SESSION_COOKIE_LIFETIME,
     // cookieDomain: process.env.SESSION_COOKIE_DOMAIN,
+    storeIdToken: false,
```

Comment:
Will we ever use id_token data, e.g. for email address? If not, if we are not grabbing the email address from auth data then it can be removed from the scope.